packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/enhanced/kg/models.py

```
@@ -30,46 +29,166 @@ class RelationshipType(Enum):
-    def __post_init__(self):
-        if self.id is None or self.id == "":
-            # Deterministic UUIDv5 id derived from stable inputs (node_type + title)
-            # Fixed namespace ensures stable generation across runs/processes
-            node_namespace = uuid.UUID("8f5b3e78-9b8c-4f07-b54e-27f7ac1d5a11")
-            name = f"{self.node_type.value}::{self.title}"
-            deterministic_uuid = uuid.uuid5(node_namespace, name)
-            self.id = f"{self.node_type.value}_{deterministic_uuid}"
+    # -------- Extra metadata --------
+    metadata: dict[str, Any] = field(default_factory=dict)
 
+    def __init__(
```

Ruff (0.15.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)